install/resources/upgrade50.php

```diff
@@ -35,3 +35,34 @@ function upgrade50_dbchanges()
 	$output->print_contents("<p>Click next to continue with the upgrade process.</p>");
 	$output->print_footer("50_done");
```

Comment:
```suggestion
	$output->print_footer("50_verify_adminemail");
```

Reply:
You've marked this comment as resolved @WhiteNeo but not made the changes? There should be a commit suggestion below the comment that will make the change for you. 😄

Reply:
I see, i have used edit button and changed the files but i was thinked on press it takes automatically your sugestions, that why i have pressed the button 😂